### ISSUE TITLE
Make minimap unused area different colored from not revealed

### ISSUE THREAD

**Issue by Szunti** [Andrettin/Wyrmsun#10]
If the map is not square then some of the minimap area is unused. Although you can send units there. It would be nice imo if it wasn't the same black as the not revealed terrain, so I could know the shape of the map. That helps when sending the first scouts.

**Comment by Andrettin** [comment]
Good point. Do you have any suggestions for the alternative color it should have instead?

**Comment by Szunti** [comment]
I like the colors of wyrmsun, I am sure you can choose a nice color or pattern, background image, I don't even know what is possible without coding too much.

**Comment by Andrettin** [comment]
Thanks :) A background image would require commissioning more interface artwork; another color would be the way to go I think.

**Comment by Kenji-03** [comment]
Suggestion 1: Use the "parchment paper" background image texture from the loading screen, so the mini-map looks like a literal map (if you give me the pixel dimensions of the mini map art asset, I can edit the background image parchment to fit).

Suggestion 2: Transparent color, so we can see the pre-existing background of whatever our interface menu's colors are (depending on if we have the cosmetic DLC)
